### ISSUE TITLE
README recommendations for EXPCUTOFF don't seem to match code.

### ISSUE THREAD

**Issue by glowthrower** [sunqm/libcint#56]
I'm unclear about how to properly set `env[PTR_EXPCUTOFF]` to achieve the desired results in my code.

The README states:

> Small integrals (< 1e-18 by default) are set to 0. If they are used in Schwarz inequality to estimate upper limit of an integral, the default integral cutoff might not be accurate enough. It can be adjusted by the parameter env[PTR_EXPCUTOFF] (since libcint 4.0). This parameter needs to be set to log(cutoff_threshold).

I read that to mean that, if I wanted to apply a Schwarz threshold of, say, 1.0e-14, I would need to set `env[PTR_EXPCUTOFF] = log(1.0e-14) \approx -32.236`.

However, looking in the code, in [`cint_const.h`](https://github.com/sunqm/libcint/blob/5de7470b5085a8c9533c891e65c0771e7a951e1f/src/cint_const.h#L104), I see:
```c
#define EXPCUTOFF       60
#ifndef MIN_EXPCUTOFF
// ~ 1e-15
#define MIN_EXPCUTOFF   40
#endif
```
Then, in (for example) [`g2c2e.c`](https://github.com/sunqm/libcint/blob/c6cf3679b1f05d12e16e113b801054c5938e14b8/src/g2c2e.c#L45), these constants are used like
```c
if (env[PTR_EXPCUTOFF] == 0) {
    envs->expcutoff = EXPCUTOFF;
} else {
    envs->expcutoff = MAX(MIN_EXPCUTOFF, env[PTR_EXPCUTOFF]);
}
```
If I'm reading this right, this means that the only meaningful setting of `env[PTR_EXPCUTOFF]` that would change behaviour is one such that `env[PTR_EXPCUTOFF] > MIN_EXPCUTOFF == 40`. That would imply a threshold of *greater than* `2.35+17`, which surely can't be right.

Should the setting perhaps be the *absolute value* of `log(cutoff_threshold)`?

**Comment by sunqm** [comment]
Yes, abs() is needed. When comparing the expcutoff, the condition is abs(gto exponents) > abs(expcutoff).

**Comment by glowthrower** [comment]
I'm not sure I quite understand that last. Do you mean that I should also be explicitly considering the exponents in whatever basis set I'm using when setting `env[PTR_EXPCUTOFF]`? Or am I safe to just set the value based upon my Schwarz threshold only?

**Comment by sunqm** [comment]
For example, if you would like to drop integrals smaller than 1e-15, you can set
```
env[PTR_EXPCUTOFF] = 34.54  # abs(log(1e-15))
```

But when you need to generate integrals for schwarz condition [(ij|ij) (kl|kl)]^{-1}, it's recommended to set it to 67.08  which is about abs(log(1e-15 ** 2)). This is due to the square root in the schwarz condition
